src/game/objects/general/bridge.c

```diff
@@ -73,20 +76,51 @@ static int32_t Bridge_GetOffset(
     return offset;
 }
 
+static void Bridge_FixEmbedded(int16_t item_num)
+{
+    ITEM_INFO *item;
+    FLOOR_INFO *floor;
+    int32_t x, y, z;
+    int16_t room_number, floor_height, old_anim, old_frame, bridge_height;
+    int16_t *bounds;
```

Comment:
Please move all variables to their usage

Reply:
Yes I don't like this style either. I based it on bat.c which does this.